### PR TITLE
[locust] Update Readme, url and logo

### PR DESCRIPTION
**Pull request by so0k**
Add url and logo to chart description

### CODE REVIEW

**Comment by so0k** [comment]
I just noticed the README still mentions this chart being in `incubator` - but it was already moved to `stable`

File: stable/locust/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 name: locust
 description: A chart for locust distributed load testing
```
Comment (prydonius) [inline stable/locust/Chart.yaml]: I updated this to "A modern load testing framework" from the website, but up to you. As long as we remove the "A chart for" part, anything is fine.

**Review by prydonius — CHANGES_REQUESTED** [review]
@so0k will make the changes described in https://github.com/kubernetes/charts/pull/891/files#r109957618

**Comment by prydonius** [comment]
> I just noticed the README still mentions this chart being in incubator - but it was already moved to stable

Yes, please update the readme to reference the stable repo too in this PR.

**Comment by so0k** [comment]
rebased and updated to include the changes requested

**Comment by so0k** [comment]
@prydonius - let me know if there is anything still missing

**Review by prydonius — APPROVED** [review]

**Comment by prydonius** [comment]
LGTM, thanks a lot @so0k!

**Pull request merged**